packages/service-library/src/servicelib/socketio_utils.py

```diff
@@ -0,0 +1,43 @@
+""" Common utilities for python-socketio library
+
+
+NOTE: we intentionally avoided importing socketio here to avoid adding an extra dependency at
```

Comment:
is that how you do it?? that is very naughty...

Reply:
@sanderegg do you prefer to add socketio everywhere? :-)